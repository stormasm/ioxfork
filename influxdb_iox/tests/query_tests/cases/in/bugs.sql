-- IOX_SETUP: Bugs

-- https://github.com/influxdata/influxdb_iox/issues/7644
-- internal error in select list
SELECT id FROM checks WHERE id in ('2','3','4','5');

SELECT id FROM checks WHERE id in ('1', '2','3','4','5');
