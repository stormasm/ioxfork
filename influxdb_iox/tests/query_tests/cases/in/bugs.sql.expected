-- Test Setup: Bugs
-- SQL: SELECT id FROM checks WHERE id in ('2','3','4','5');
+----+
| id |
+----+
+----+
-- SQL: SELECT id FROM checks WHERE id in ('1', '2','3','4','5');
+----+
| id |
+----+
| 1  |
| 1  |
| 1  |
| 1  |
+----+